doc/user-managerd.qdocconf

```diff
@@ -1,8 +1,8 @@
-project         = Sailfish User Manager Daemon
+project         = user-managerd
 description     = Sailfish User Manager Daemon Reference Documentation
 versionsym      =
 version         = 0.8.0
-url             = https://github.com/sailfishos/user-managerd/
+url             = $BASE_URL/user-managerd
```

Comment:
Local build fails due to undefined env variable
```
In file included from /usr/share/sailfish-qdoc-template/offline.qdocconf:3:
<path>/jolla/user-managerd/doc/user-managerd.qdocconf:5: error: Environment variable 'BASE_URL' undefined
Environment variable 'BASE_URL' undefined
```
Build succeeds when building as follows:
> BASE_URL=https://github.com/sailfishos mb2 --no-snapshot  -t SailfishOS-aarch64 -X build

Is it intentional that BASE_URL needs to be defined? Does this affect to OBS?

Reply:
You are missing https://github.com/sailfishos/sailfish-qdoc-template/pull/2.

try this instead (including the third related PR):

```
mkdir ~/tmp/RPMS
alias mb2='mb2 --output-prefix ~/tmp/RPMS --task'
cd sailfish-qdoc-template
git checkout jb53870
mb2 -t SailfishOS-aarch64 build
cd ../user-managerd
git checkout jb53870
mb2 -t SailfishOS-aarch64 build
cd ../base-sailfish-mdm
git checkout jb53870
mb2 -t SailfishOS-aarch64 -s rpm/sailfish-mdm.spec build
```

This way you get the packages built locally under the same task deployed under the build environment automatically.

And note that in order to build sailfish-mdm you will also need to unset `rpm.install.excludedocs` under /etc/zypp/zypp.conf - it needs the files installed by user-managerd-doc. Do that under the original build target:

```
sdk-assistant -y maintain SailfishOS-aarch64 sed -i -e 's/^\s*rpm.install.excludedocs/#&/' /etc/zypp/zypp.conf
```

This is not needed on OBS and I am working on a solution for SDK.

Reply:
Regarding OBS I was also referring to the env variable. After checking the https://github.com/sailfishos/sailfish-qdoc-template/pull/2 all makes more sense :-) :+1: